applications/rag/README.md

```diff
@@ -36,6 +36,17 @@ gcloud container clusters create-auto ${CLUSTER_NAME:?} \
   --release-channel rapid
 ```
 
+### Enable [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect)
+
+Note: Skip this step if setting `create_network = true`
```

Comment:
Or mention in terms of prerequisites for an existing network are.

Reply:
Done (added to header)